src/main/perl/Path.pm

```diff
@@ -346,6 +353,21 @@ sub any_exists
     return $path && (-e $path || -l $path);
 }
 
+=item is_symlink
+
+Test if C<path> is a symlink.
+
+Returns true as long as C<path> is a symlink, even though the 
```

Comment:
"even though" would normally be used when dealing with a non-conditional event. e.g. "The politician decided to vote against the law even though his or her party was for it." Whereas in this case it's not a given that the symlink does not exist so if or when would be more appropriate.

```Returns true as long as C<path> is a symlink, even if the symlink target does not exist.``` or
```Returns true as long as C<path> is a symlink, including when the symlink target does not exist.```